griptape/tools/file_manager/tool.py

```diff
@@ -24,7 +25,7 @@ class FileManager(BaseTool):
     """
 
     workdir: str = field(default=Factory(lambda: os.getcwd()), kw_only=True)
-    default_loader: BaseLoader = field(default=Factory(lambda: FileLoader()))
+    default_loader: BaseLoader = field(default=None, kw_only=True)
```

Comment:
`BaseLoader` should be `Optional[BaseLoader]`

Reply:
Done